views/includes/head.hbs

```diff
@@ -30,10 +30,15 @@
 <link rel="stylesheet" type="text/css" href="/css/style.css">
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
 
+<!-- Origin Trial Token, feature = Web Share, origin = https://pwa-directory.appspot.com, expires = 2016-12-06 -->
+<meta http-equiv="origin-trial" data-feature="Web Share" data-expires="2016-12-06" content="AuNW4MmMk4aT1N9HPDpSlUPkBywNReGpXpbTC+9nOE/rv6W6mToe7J1ZSmz35g+6BbPzjmE9gpBTZOrb2aXf6gwAAABgeyJvcmlnaW4iOiAiaHR0cHM6Ly9wd2EtZGlyZWN0b3J5LmFwcHNwb3QuY29tOjQ0MyIsICJmZWF0dXJlIjogIldlYlNoYXJlIiwgImV4cGlyeSI6IDE0ODEwNDQyMDJ9">
+
 <script id="config" type="application/json">
 {{{configstring}}}
 </script>
 <script src="/js/gulliver.js" async defer></script>
+<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js" defer></script>
```

Comment:
I cannot change it, because it needs the order to load Google charts before running the script. How could we fix that?